packages/api/cms-api/src/mailer/mailer.service.ts

```diff
@@ -33,25 +41,58 @@ export class MailerService {
      * @param mailTypeForLogging Mail type, e.g. order confirmation, order cancellation, etc. to filter in the mailer log
      * @param additionalData Put your additional data here, e.g. orderId, resourcePoolId, etc.
      * @param originMailOptions `from` defaults to this.config.mailer.defaultFrom, sendAllMailsBcc is always added to `bcc`
+     * @param logMail When set to false, the email will not be logged to the database.
```

Comment:
as we have an explicit `mailTypeForLogging`, you could also set that to `false` to disable logging intstead of adding a new option

Reply:
IMO adding an additional option is more intuitive here. As a dev I'd have to know that setting mailTypeForLogging needs to be false to disable logging (I'd also need to know about logMail, but that's more straight forward).